etna/packages/etna-js/actions/magma_actions.js

```diff
@@ -68,7 +70,7 @@ export const consumePayload = (dispatch, response) => {
 };
 
 const showError = dispatch => response => {
-  if ('error' in response) {
+  if (_.isObject(response) && 'error' in response) {
```

Comment:
Curious -- why was this change necessary?  would response && 'error' in response suffice?

Reply:
The TSV response from Magma (for `/query` TSV download) is just a string, so `'error' in response` throws an exception.